### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by jodh-intel**
Added minimal README docs where missing, added TOCs, fixed headings and ultimately ensured all docs are referenced.

Fixes: #501.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
The new README's on this PR are a bit skeletal but it's a start!

File: static-build/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Static build
+
+This directory contains files to build static versions of Kata Container.
```
Comment (devimc) [inline static-build/README.md]: nit: Container**s**

Comment (jodh-intel) [reply static-build/README.md]: Good catch! :smile: Fixed.

**Comment by jodh-intel** [comment]
Noooooooooo.....

```
Kernel directory has changes check kernel/kata_config_version changed
ERROR: Please bump version in kernel/kata_config_version
```

**Pull request merged**